test/FromModelica/ExtendsClause-1.html

```diff
@@ -0,0 +1,251 @@
+    list-style-type: none;
+  }
+  h1{
+    font-size: 2em;
+  }
+  .h2, h2{
+    font-size: 1.7em;
+  }
+  .h3, h3{
+    font-size: 1.5em;
+  }
+  .h4, h4{
+    font-size: 1.2em;
+  }
+  .footer{
+  /*  background-color: white;
+    border-color: white; */
+    background-color: #f8f8f8;
+    padding: 10px;
+    font-size: 75%;
+    border-width:0px;
+    background-image: none;
+    border-radius: 4px;
+
+  }
+
+  /* Counters for header */
+  body {counter-reset: h2}
+  h2 {counter-reset: h3}
+  h3 {counter-reset: h4}
+  h4 {counter-reset: h5}
+  h5 {counter-reset: h6}
+
+  h2:before {counter-increment: h2; content: counter(h2) ". "}
+  h3:before {counter-increment: h3; content: counter(h2) "." counter(h3) ". "}
+  h4:before {counter-increment: h4; content: counter(h2) "." counter(h3) "." counter(h4) ". "}
+  h5:before {counter-increment: h5; content: counter(h2) "." counter(h3) "." counter(h4) "." counter(h5) ". "}
+  h6:before {counter-increment: h6; content: counter(h2) "." counter(h3) "." counter(h4) "." counter(h5) "." counter(h6) ". "}
+
+  h2.nocount:before, h3.nocount:before, h4.nocount:before, h5.nocount:before, h6.nocount:before
+    { content: ""; counter-increment: none }
+
+  </style>
+
+</head>
+<body>
+
+<h2>FromModelica.ExtendsClause<a name="FromModelica.ExtendsClause"></a></h2>
+
+  <p>model with extends clause</p>
+
+<h3>Functionality</h3>
+  
+test...test...test...test...test...
+
+
+<h3>Parameters</h3>
+    <p>
+    It has the following parameters:
+    </p>
+    <table>
+    <tr><th>Name    </th>
+    <th class="description">Description</th>
+    <th>Value    </th>
+    <th>Unit     </th>
+    <th>Display unit</th>
+    <th class="type">Type        </th>
+    <th>min    </th>
+    <th>max    </th>
+    </tr>
+    <tr class="tab"><td colspan="8">General</td></tr>
+    <tr class="group"><td colspan="8">Parameters</td></tr>
+    <tr>
+    <td><code>length</code><a name="FromModelica.ExtendsClause.length"></a></td>
+    <td>Length of the pipe</td>
+    <td></td>
+    <td></td>
+    <td></td>
+    <td class="type">Modelica.SIunits.Length</td>
+    <td></td>
+    <td></td>
+    </tr>
+    </table>
+
+
+
+<h3>Blocks</h3>
+    <p>
+    It has the following blocks:
+    </p>
+    <table>
+    <tr>
+      <th class="type">Type</th>
+      <th>Name</th>
+      <th class="description">Description</th>
+      <th>Parameter Assignments</th>
+    </tr>
+    <tr>
+      <td class="type">
+      <code>
+      <a href="http:&#x2F;&#x2F;simulationresearch.lbl.gov&#x2F;modelica&#x2F;releases&#x2F;v5.0.1&#x2F;help&#x2F;Buildings_Controls_OBC_CDL_Continuous.html#Buildings.Controls.OBC.CDL.Continuous.LimPID">Buildings.Controls.OBC.CDL.Continuous.LimPID</a>
+      </code>
+      </td>
+      <td>
+      <code>
+      Buildings.Controls.OBC.CDL.Continuous.LimPID
+      </code>
+      <a name="FromModelica.ExtendsClause.Buildings.Controls.OBC.CDL.Continuous.LimPID"></a></td>
+      <td>model with extends clause</td>
```

Comment:
This is parsed incorrectly: extending a model is different from instantiating a model. The way this is documented is as if it were to instantiate `LimPID`.